### ISSUE TITLE
Display import info in NetCDF import dialog

### ISSUE THREAD

**Issue by dannyparsons** [africanmathsinitiative/R-Instat#6275]
From @rdstern 
> After going to the sub-dialogue to select specific points (station locations) to import I returned to the main dialogue. There was no message on what I was about to import when I pressed ok. Ideally as well as saying how many files it has to read, it might also say how many grid points it will import, once I press OK. Then when I return from the sub-dialogue that could change - in my case to say there will be 4 grid-points imported.

**Comment by dannyparsons** [comment]
I agree this would be nice to do. Nothing difficult but there's lots of cases to take care of so it might take longer than the benefit of doing other things so I've suggest 0.7.x milestone.

**Comment by rdstern** [comment]
ok. Agreed!
